releasenotes/notes/fix-psycopg-async-iteration-d48a62f8aa597457.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    psycopg: Fixes issue of psycopg async iterator previously not returning objects, by implementing async generator.
```

nit: We can simplify this release note
```suggestion
    psycopg: Ensures traced async cursors return an asynchronous iterator object.
```